tsconfig-base.json

```diff
@@ -3,8 +3,13 @@
 {
   "compilerOptions": {
     "composite": true,
+
+    // Set up incremental builds. All sub-projects emit under outDir, maintaining
+    // directory structure relative to rootDir. No effect on include, exclude, etc.
     "emitDeclarationOnly": true,
     "declarationMap": true,
+    "outDir": "./.tmp/tsbuildinfo/",
+    "rootDir": ".",
```

Comment:
I was looking at the `outDir`s in all the tsconfigs recently and was kind of annoyed, but it turns out if I had just looked at the `rootDir` docs I would have realized it only affects emit, not `includes`, `excludes`, etc, so specifying this once in the base tsconfig works perfectly. Makes the extending tsconfigs much nicer, and should fix your problem in https://github.com/GoogleChrome/lighthouse/pull/14054/commits/c55da8bfca79ebebd3a3bb25b6a412b9b3c285c4, @connorjclark